client/src/components/RecoverPassword.jsx

```diff
@@ -14,7 +14,7 @@ export class RecoverPassword extends Component {
         this.setState({ [id]: e.currentTarget.value })
     };
 
-    make_recover = (e) => {
+    recover_password = (e) => {
```

Comment:
Better to use camelCase in JS for naming functions and other. [ref](https://github.com/airbnb/javascript/#naming--camelCase). Also this function is not a password recovery, it's just a send a mail, so this name is not very suitable.

Reply:
Yes, or just sendMailRecoverPassword, the function's name should tell what it does.
I know, it's my bad. I will fix it in the future.

Reply:
Also I can refer to "Code Complete 2nd edition" p. 260-262. 
<blockquote>
The most important consideration in naming
 a variable is that the name fully and 
accurately describe the entity the variable represents. An effective technique for com-
ing up with a good name is to state in words what the variable represents.
</blockquote>